src/EventPage/screens/EventPageContainer.js

```diff
@@ -3,12 +3,13 @@ import EventPage from '../components/EventPage';
 
 class EventPageContainer extends React.Component {
   state = {
-    event: { title: '' },
+    event: { key: '1',
+      title: 'Creating Reality Hackthon',
```

Comment:
eventually `Date` will be coming from the server as a javascript Date object so we will have to map that to your format, but that's just a headsup